common.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python
+
+import os
+import logging
+import tarfile
+import urllib
+import subprocess
+import json
+import time
+import errno
+
+
+logging.basicConfig(level=logging.INFO)
+log = logging.getLogger("common")
+
+packet_auth_token = os.environ["PACKET_AUTH_TOKEN"]
+packet_project_id = 'b4357aca-d706-4096-9e44-0a7c69d40ebe'
```

Comment:
This should be a environment variable. And many things defined here should be able to override by a environment variable, e.g. machine type/os/zone/project etc.